### PR TITLE
Object instantiation update

### PR DESCRIPTION
**Pull request by almargit**

### CODE REVIEW

File: learn/classes/object_instantiation.md [hunk 1]
```diff
@@ -12,17 +12,75 @@ A class constructor in Python is a special method that is executed when an objec
...
+
+# Instantiate a Recipe object
+my_recipe = Recipe("Spaghetti Bolognese", ["spaghetti", "tomato sauce", "ground beef"])
+
+# Access the object's attributes
+print("Recipe Name:", my_recipe.name)
+print("Ingredients:", my_recipe.ingredients)
+```
+
+In the above example, the `Recipe` class has a constructor that sets the attributes `name` and `ingredients` for each new object that is instantiated. The `my_recipe` object is instantiated with the name "Spaghetti Bolognese" and a list of ingredients. The print statements will output "Recipe Name: Spaghetti Bolognese" and "Ingredients: ['spaghetti', 'tomato sauce', 'ground beef']."
```
Comment (angarsky) [inline learn/classes/object_instantiation.md:28]: Highlight this output as a code : "Recipe Name: Spaghetti Bolognese" and "Ingredients: ['spaghetti', 'tomato sauce', 'ground beef']."

File: learn/classes/object_instantiation.md [hunk 2]
```diff
@@ -12,17 +12,75 @@ A class constructor in Python is a special method that is executed when an objec
...
+class Bike(Vehicle):
+    def __init__(self, make, model, wheel_count):
+        super().__init__(make, model)
+        self.wheel_count = wheel_count
+```
 
-p = Person("Jane", 25)
-print("Name:", p.name)
-print("Age:", p.age)
+### Abstract Base Classes
```
Comment (angarsky) [inline learn/classes/object_instantiation.md:63]: As I remember we have the separated page about Abstract classes.I guess we can remove this section. Or write a short description + provide a link.

File: learn/classes/object_instantiation.md [hunk 3]
```diff
@@ -12,17 +12,75 @@ A class constructor in Python is a special method that is executed when an objec
...
+        self.color = color
+
+class Circle(Shape):
+    def __init__(self, color, radius):
+        super().__init__(color)
+        self.radius = radius
 ```
 
-In the above example, the `Person` class has a constructor that sets the attributes `name` and `age` for each new object that is instantiated. The `p` object is instantiated with the `name` of ``Jane`` and the `age` of `25`. The `print` statements will output `Name: Jane` and `Age: 25`.
+Conclusion
```
Comment (angarsky) [inline learn/classes/object_instantiation.md:81]: Conclusion should be a heading. But it's not a conclusion of the whole page. Do we need it really?

**Review by angarsky — CHANGES_REQUESTED** [review]
Good job! But I found a couple of issues, as well :)

File: learn/classes/object_instantiation.md (hunk 2 above)
Comment (almargit) [reply learn/classes/object_instantiation.md:63]: done

File: learn/classes/object_instantiation.md (hunk 3 above)
Comment (almargit) [reply learn/classes/object_instantiation.md]: deleted

**Review by angarsky — CHANGES_REQUESTED** [review]
Please, update the Abstract class section.

**Review by angarsky — APPROVED** [review]
Thank you! Approved.

**Pull request merged**